src/python/doc/rips_complex_ref.rst

```diff
@@ -12,3 +12,54 @@ Rips complex reference manual
    :show-inheritance:
 
    .. automethod:: gudhi.RipsComplex.__init__
+
+======================================
+Weighted Rips complex reference manual
+======================================
+
+.. autoclass:: gudhi.WeightedRipsComplex
+   :members:
+   :undoc-members:
+   :show-inheritance:
+
+   .. automethod:: gudhi.WeightedRipsComplex.__init__
+
+Basic examples
+-------------
```

Comment:
sphinx likes the underline to match the size of the title exactly, so one extra `-` I think.